keep/common/management/commands/update_pidman_title.py

```diff
@@ -0,0 +1,247 @@
+        audio_objects = self.repo.get_objects_with_cmodel(AudioObject.AUDIO_CONTENT_MODEL)
+        self.stderr.write("%i Audio objects found." % len(audio_objects))
+
+        video_objects = self.repo.get_objects_with_cmodel(Video.VIDEO_CONTENT_MODEL)
+        self.stderr.write("%i Video objects found." % len(video_objects))
+
+        arrangement_objects = self.repo.get_objects_with_cmodel(ArrangementObject.ARRANGEMENT_CONTENT_MODEL)
+        self.stderr.write("%i Arrangement objects found." % len(arrangement_objects))
+
+        diskimage_objects = self.repo.get_objects_with_cmodel(DiskImage.DISKIMAGE_CONTENT_MODEL)
+        self.stderr.write("%i DiskImage objects found." % len(diskimage_objects))
+
+        simple_collection_objects = self.repo.get_objects_with_cmodel(SimpleCollection.COLLECTION_CONTENT_MODEL)
+        self.stderr.write("%i SimpleCollection objects found." % len(simple_collection_objects))
+
+        collection_objects = self.repo.get_objects_with_cmodel(CollectionObject.COLLECTION_CONTENT_MODEL)
+        self.stderr.write("%i Collection objects found." % len(collection_objects))
+
+        mailbox_objects = self.repo.get_objects_with_cmodel(boda.Mailbox.MAILBOX_CONTENT_MODEL)
+        self.stderr.write("%i Mailbox objects found." % len(mailbox_objects))
+
+        # end the object collection process and start
+        self.stdout.write("Object collection finished.")
+
+        # update objects in each content model
+        self.update_progress(audio_objects, "AudioObject", self.pidman)
+        self.update_progress(video_objects, "VideoObject", self.pidman)
+        self.update_progress(arrangement_objects, "ArrangementObject", self.pidman)
+        self.update_progress(diskimage_objects, "DiskImage", self.pidman)
+        self.update_progress(simple_collection_objects, "SimpleCollection", self.pidman)
+        self.update_progress(collection_objects, "CollectionObject", self.pidman)
+        self.update_progress(mailbox_objects, "Mailbox", self.pidman)
+
+    def update_progress(self, objects, task_name, pidman):
+        """Update the objects in Pidman and reports progress back to the user.
+
+        Args:
+           objects (array): Array of objects retreived from Fedora
+           task_name (str): Name of each task (object collection)
+           pidman (DjangoPidmanRestClient): PidmanRestClient that can be used
+            to interact with Pidman
+        """
+        # update progress on the screen
+        sys.stdout.write("Starting %s tasks \n" % task_name)
+        sys.stdout.flush()
+
+        # initialize counters
+        total_count = len(objects)
+        fail_count = 0
+        change_count = 0
+        nochange_count = 0
+
+        # create log files
+        change_log = open(("%s/%s" % (self.output_path, "change.csv")), "w+")
+        nochange_log = open(("%s/%s" % (self.output_path, "nochange.csv")), "w+")
+        summary_log = open(("%s/%s" % (self.output_path, "summary.log")), "w+")
```

Comment:
Seems complicated to have so many different output files, maybe they could be consolidated somehow?